### DISCUSSION TITLE
What is the `@hint` syntax highlighting capture for?

### DISCUSSION THREAD

**Discussion by valentinegb** [zed-industries/zed#19143] [Feature Requests]
I couldn't find any instances of it being used from a code search of Zed or any extensions in @zed-extensions, so I'm not sure what exactly it's intended to be used for. I was thinking, if I'm to interpret it just as "a hint", then maybe we could standardize something like `@hint.note`, `@hint.tip`, `@hint.important`, `@hint.warning`, and `@hint.caution` (based on [GitHub's blockquote headers](https://github.com/orgs/community/discussions/16925)) to allow for things like distinct highlights for log levels in @nervenes's [zed-log](https://github.com/nervenes/zed-log) extension. It currently uses [some non-standard captures](https://github.com/nervenes/zed-log/blob/main/languages/log/highlights.scm).

**Comment by thataboy** [comment]
It's used for things that appear in-line in the editor, like git blame, inlay hints (for function parameters, etc), diagnostic messages, probably other things.

**Reply by valentinegb** [reply]
Thanks, I think I'll make an issue for my proposal, then.
